test/tests.jl

```diff
@@ -145,6 +155,11 @@ function testFile5(inputData)
     @test inputData.swellIndex[2] == 0.10
     @test inputData.compressionIndex[1] == 0.25
     @test inputData.heaveActiveZoneDepth == 8.00
+    P, PP, x = OutputFormat.performGetCalculationValue(outputData)
+    @test P[1] != PP[1]
+    @test P[2] != PP[2]
+    @test P[3] != PP[3]
+    @test P[4] != PP[4]
```

Comment:
You test that the values are not equal to each other, but do you test that the values themselves are correct?